rosa_tutorials/rosa-mobb-cli-quickstart.adoc

```diff
@@ -0,0 +1,271 @@
+:_content-type: ASSEMBLY
```

Comment:
It's commented out so it won't build, but since I started bringing it over and converting it to ascii before we pivoted to the ALB doc, I didn't want to lose the progress